src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerController.java

```diff
@@ -53,4 +61,36 @@ public CaseEventHandlerController(List<CaseEventHandler> handlerServices) {
 
     }
 
+    @ApiOperation("Handles the CCD case event message")
+    @ApiResponses({
+        @ApiResponse(
+            code = 204,
+            message = "Message processed successfully",
+            response = Object.class)
+    })
+    @PostMapping(path = "/messages/{message_id}", consumes = {MediaType.APPLICATION_JSON_VALUE})
```

Comment:
Was wondering if this should be a `PUT` so we could use it for updating as well.  I think this is fine though we could it later.

Reply:
Let's do it as a part of another ticket.

Reply:
sure.